src/components/UserProfile/ScheduleReasonModal/ScheduleReasonModal.jsx

```diff
@@ -29,7 +29,7 @@ const ScheduleReasonModal = ({
     const initialFetching = async () => {
       fetchDispatch({ type: 'FETCHING_STARTED' });
       const response = await getReasonByDate(userId, date);
-      console.log(response);
+      // console.log(response);
```

Comment:
It is generally a good idea to not commit `console.log()` statements. They are helpful when used in your local development environment, but serves no purpose in the actual app and can even cause security issues. Additionally, it can also be forgotten and cause tech debt to accumulate over time.

Reply:
Hello @rusalshrestha98, thank you for taking the time to review this PR. I will go ahead and remove the unnecessary logs and try to resolve any conflicts in the branch.